jetty-server/src/main/config/etc/jetty.xml

```diff
@@ -126,6 +126,6 @@
     <Set name="stopTimeout"><Property name="jetty.server.stopTimeout" default="5000"/></Set>
     <Set name="dumpAfterStart"><Property name="jetty.server.dumpAfterStart" deprecated="jetty.dump.start" default="false"/></Set>
     <Set name="dumpBeforeStop"><Property name="jetty.server.dumpBeforeStop" deprecated="jetty.dump.stop" default="false"/></Set>
-    <Set name="openNetworkConnectors"><Property name="jetty.server.openNetworkConnectors" default="false"/></Set>
+    <Set name="verifiedStartSequence"><Property name="jetty.server.verifiedStartSequence" default="false"/></Set>
```

Comment:
How many versions of jetty have had the older name?
Do we want to break this kind of backward compat in 9.4.x mid-stream?  or wait for jetty 10.x?

Reply:
none - the "old" name was new in this pull request.